Client/Frontend/Browser/Authenticator.swift

```diff
@@ -7,7 +7,7 @@ import Shared
 import Storage
 import Deferred
 
-private let CancelButtonTitle = NSLocalizedString("Cancel", comment: "Authentication prompt cancel button")
+private let CancelButtonTitle = NSLocalizedString("Cancel", comment: "Label for Cancel button")
```

Comment:
The idea is to fix only comments, so it can eventually be ported to v5.x if we want. It's also not safe for me to touch more than the basic stuff in swift code ;-)